test/suites/dev/common/test-xcm-v5/test-mock-hrmp-transact-ethereum-6.ts

```diff
@@ -109,7 +109,7 @@ describeSuite({
         let expectedTransferredAmount = 0n;
         let expectedTransferredAmountPlusFees = 0n;
 
-        const targetXcmWeight = 30_000_000_000n + 100_000_000n;
+        const targetXcmWeight = 30_000_000_000n + 700_000_000n;
```

Comment:
What is the reason for this value to increase 7 times?

Reply:
It's not necessarily that part of the cost increasing 7x. It might be the `30_000_000_000n` increasing by a smaller factor. Across all tests it's generally difficult to tell what are the cost components for a fee. As you suggested, I will try to define some constants for them once the test suites run all green.